pgcli/pgcompleter.py

```diff
@@ -41,8 +41,13 @@ def __init__(self, smart_completion=True, pgspecial=None, settings=None):
         self.smart_completion = smart_completion
         self.pgspecial = pgspecial
         self.prioritizer = PrevalenceCounter()
-        self.asterisk_column_order = (settings or {}).get(
+        settings = settings or {}
+        self.casing_file = settings.get('casing_file')
+        self.generate_casing_file = (
+            settings.get('generate_casing_file') == 'True')
```

Comment:
I think the conversion from the string 'True' to a boolean needs to happen as early as possible, when PGCli.__init__ pulls it out from the config option. The pgcompleter settings struct should expect a boolean value here